### ISSUE TITLE
AttributeError: 'SplitterWindow' object has no attribute 'SetSashSize'

### ISSUE THREAD

**Issue by jnthngdbt** [belangeo/zyne#6]
Hi,

I am having fun with pyo and would like to try out Zyne.

However, I get this error when running it on Windows (`python Zyne.py`):

```
Traceback (most recent call last):
  File "Zyne.py", line 749, in OnInit
    self.frame = ZyneFrame(None)
  File "Zyne.py", line 222, in __init__
    self.splitWindow.SetSashSize(0)
AttributeError: 'SplitterWindow' object has no attribute 'SetSashSize'
OnInit returned false, exiting...
```

- pyo 1.0.3
- wxPython 4.1.1

Thanks!

**Comment by belangeo** [comment]
Hi, just pushed a couple of fixes for WxPython 4.1.

**Comment by jnthngdbt** [comment]
Hi, this is great and thanks to you.
